web/src/components/columnGrid.js

```diff
@@ -24,16 +24,20 @@ const Heading = styled.h2`
         width: 3rem;
     }
 `;
+const Description = styled.p`
+    ${tw`text-black`}
+`;
 
 const Grid = styled.div`
     ${tw`grid grid-cols-1 lg:grid-cols-3 gap-24 lg:px-16`}
 `;
 
 export default function ColumnGrid({ content }) {
-    const { heading, indexIndustries } = content;
+    const { description, heading, indexIndustries } = content;
     return (
         <Section>
             <Heading>{heading}</Heading>
+            <Description>{description}</Description>
```

Comment:
Will every page that uses this component have a description?

Reply:
@asasmith no but it will only show up once we input the content in sanity

Reply:
but there's no conditional logic here. So something will always be present, right?

Reply:
@asasmith No thats how I have done it in other components on silverado. Itll only show up if you set some object to the description

Reply:
If you look on the index page you'll see that an empty `<p>` tag is added to the dom. I think we need to have conditional logic to not create dom nodes if there isn't content there. I'm pretty sure that's the pattern we used on Silverado. If it's not then I'm sorry that it's not something I caught.